specification/examples/organisation_v1.json

```diff
@@ -0,0 +1,6923 @@
+{
```

Comment:
Is this copy pasted from somewhere. Glazing over looking at this one.

Reply:
This is live output from the API.